Dockerfile

```diff
@@ -26,7 +26,7 @@ COPY --chown=sydent:sydent ["sydent", "sydent"]
 COPY --chown=sydent:sydent ["README.rst", "pyproject.toml", "poetry.lock", "./"]
 
 # Install dependencies
-RUN python -m poetry install --no-dev --no-interaction --extras "prometheus sentry"
+RUN python -m poetry install -vv --no-dev --no-interaction --extras "prometheus sentry"
```

Comment:
Do you still want the `-vv` here?

Reply:
My thinking was to leave it in as it can't do any harm.